componentapiexample/src/main/java/net/gini/android/vision/component/MainActivity.java

```diff
@@ -175,12 +174,12 @@ private void startGiniVisionCompat() {
                     @Override
                     public void permissionGranted() {
                         // NOTE: on Android 6.0 and later the camera permission is required before checking the requirements
-                        final RequirementsReport report = GiniVisionRequirements.checkRequirements(
-                                MainActivity.this);
-                        if (!report.isFulfilled()) {
-                            showUnfulfilledRequirementsToast(report);
-                            return;
-                        }
+//                        final RequirementsReport report = GiniVisionRequirements.checkRequirements(
```

Comment:
If you don't want it, delete it 🗑

Reply:
Good catch. I commented it out to allow running on emulators. I will uncomment it and only show unfulfilled requirements without blocking execution of the GVL.